antora.yml

```diff
@@ -1,14 +1,14 @@
 name: spark
 title: Neo4j Spark
-version: '5.0'
+version: '5'
 start_page: ROOT:index.adoc
 nav:
 - modules/ROOT/nav.adoc
 
 asciidoc:
   attributes:
     theme: docs
-    connector-version: '5.0.3'
+    connector-version: '5'
```

Comment:
I believe here you can leave it annotated with the minor version, so the homepage will inform that

Reply:
So this is actually a shortcut to add the version of the connector automatically to the index page, as you can check here https://github.com/neo4j/docs-spark/blob/dev/modules/ROOT/pages/index.adoc?plain=1 
If you want to be specific about the minor version, then that's the place